### ISSUE TITLE
User should be able to quickly jump to unread channels, if they exist

### ISSUE THREAD

**Issue by holmesworcester** [TryQuiet/quiet#1101]
#544 is missing an important feature that Slack has. 

When the user has unread messages, the channels or the DM thread that contains them should be listed here instead of recent channels, so that users can jump to unread threads quickly with the keyboard. 

In the case where messages are unread, "Recent Channels" in our design can change to "Unread Messages", and the list of reecnt channels should change to list all channels with unreads. 

The first channel in the list should be selected by default, so that the user can use "Ctrl-K + enter" to cycle through each channel with unread messages. If the user starts typing, the UI should switch to be the same as channel search, so they can switch to any channel not just ones with unread messages.

(I'll create a separate issue to note that DMs should display first, here, but we don't need to worry about that now because we don't have DMs working yet.)

**Comment by kingalg** [comment]
Version: 1.1.0-alpha.3
 CTRL+k is working, but I need specific informations about how it should work. @holmesworcester @Kacper-RF I will list how it works now and please let me know if this is or is not expected behaviour:

1. Shortcut for specific os: 
a) Linux CTRL+k
b) Windows CTRL+k
c) MacOs Command+k
2. If there is any channel with unread messages the only list that will be displayed is list "UNREAD MESSAGES"
<img width="479" alt="Screenshot 2023-03-13 at 16 13 15" src="https://user-images.githubusercontent.com/68909168/224744474-7a36e03e-b689-4295-8f91-c55d74538d5a.png">
3. If all messages are left it will display list of "RECENT CHANNELS"
<img width="475" alt="Screenshot 2023-03-13 at 16 15 13" src="https://user-images.githubusercontent.com/68909168/224745017-0367bcec-d299-4094-a4b7-c3dce9c7c1a2.png">
4. Only 3 recent channels will be displayed (What's the rule for them? Is it channel recently used by user? By community? What does it means "used" - opened, message send...?).
5. List of all channels with unread messages. Up to how many? I think that we have potential **bug** here. I found no way to scroll this list. If I make Quiet window smaller it shows only part of the list and I can't scroll it. Below you can see the same list on different size windows:
<img width="596" alt="Screenshot 2023-03-13 at 16 22 53" src="https://user-images.githubusercontent.com/68909168/224747134-605db87d-9427-4b5d-98c5-d58057fc0796.png">
<img width="1033" alt="Screenshot 2023-03-13 at 16 25 27" src="https://user-images.githubusercontent.com/68909168/224747848-232441a7-791d-4b50-8d95-2e6be873ae73.png">
6. Channel can be search by any letter or number in it's name.
7. If user looks for a channel with a name/number that is absent in all channels user still gets a result. Why? Based on what? 
<img width="623" alt="Screenshot 2023-03-13 at 16 31 08" src="https://user-images.githubusercontent.com/68909168/224749453-64884016-7463-42ab-9aed-5f592f761e16.png">
8. When the user choose a channel from the list he is directed to this channel (it's smooth). 

Please let me know if this is correct behaviour and if I'm missing anything that should be added to my checklist.

**Comment by kingalg** [comment]
Above comment actually fits better to issue #544 (that I saw only after checking this one) and comments in this issue answered most of my questions but let me know if my list sums it up correctly.

**Comment by holmesworcester** [comment]
I think the list should be scrollable and that X close button should not wrap when font size is bigger. Other than that this looks good to me.

**Comment by Kacper-RF** [comment]
@kingalg 
I'm also adding here a link to Figma with requirements comments on screens - https://www.figma.com/file/FHbtDnmYUHj9mOPzVIWaQg/Keyboard-shortcuts

I will take care of problem with scroll and X button because they're definitely bugs

**Comment by kingalg** [comment]
Version: 1.1.0-alpha.6

This part is working really nicely right now.
